src/app/payment/reservation-content.tsx

```
@@ -0,0 +1,479 @@
+        <div className="space-y-4">
+          <h2 className="text-xl font-semibold">
+            {language === "en" ? "Your Information" : "Tu Información"}
+          </h2>
+          <div className="bg-white rounded-xl p-4 border border-gray-100 shadow-sm">
+            <div className="flex items-start gap-4">
+              <div className="p-2 bg-blue-50 rounded-full">
+                <User2 className="h-6 w-6 text-blue-500" />
+              </div>
+              <div className="flex-1">
+                <h3 className="font-medium">
+                  {consumerData?.consumer?.firstName + " " + consumerData?.consumer?.lastName || (language === "en" ? "Unknown User" : "Usuario Desconocido")}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)